src/types/types.ts

```diff
@@ -8,3 +8,17 @@ export interface LogoPartnersType {
   alt?: string;
   width?: number;
 }
+
+export interface ImageCarouselType {
+  src: string;
```

Comment:
mungkin pakai `?` semua saja keknyaa kayak ```key?: null | undefined;```, untuk menghindari error kalau tidak dipanggil key nya.

Reply:
okay. how about this 16ae384c032a789f48f6bee659cee8b698a2677c kak? i added question mark to the all properties there

Reply:
boleh mut